README.md

```diff
@@ -1,9 +1,39 @@
 # jme-validation
 
-Setup based on Bamboo (https://gitlab.cern.ch/cp3-cms/bamboo) to validate JME diliverables
+Framework based on Bamboo (https://gitlab.cern.ch/cp3-cms/bamboo) to validate JME deliverables.
 
-you need to create the file `bamboo.init`
-with the folling information:
+## Installation
+In order to install the framework, first git-clone:
+```
+git clone https://github.com/cms-jet/jme-validation
+```
+Then, run the installation script:
+```
+source install.sh
+
+```
```

Comment:
do we want to add that the installation script is installing bamboo, plotit and CMSJME caluclator? Just to be sure that people do not install it on their own?